src/game.cpp

```diff
@@ -14134,34 +14134,34 @@ void game::wait()
 
     switch (as_m.ret) {
     case 1:
-        time = 5000;
+        time = 100 * MINUTES( 5 );
```

Comment:
Since it's `time` (I mean not `moves`), it would be more consistent (and less verbose) to multiply only once:
```c++
player_activity new_act( actType, 100 * time, 0 );
```
Also, if you extract the `switch` into a function, you would save x10 `break`s and get a meaningful name:
```c++
...
case 5:
    return HOURS( 3 );
case 6:
    return HOURS( 6 );
...
```